src/mlite6.pc.in

```diff
@@ -1,11 +1,11 @@
 prefix=/usr
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
-includedir=${prefix}/include/mlite
```

Comment:
Don't we still need mlite.pc.in for non qt6 builds?

Reply:
didn't notice we still have mlite5, nevermind then!

Reply:
We do need to keep this.

Reply:
return